workflows/prognostic_c48_run/runtime/steppers/machine_learning.py

```diff
@@ -162,12 +163,29 @@ def predict(model: MultiModelAdapter, state: State) -> State:
     return {key: cast(xr.DataArray, output[key]) for key in output.data_vars}
 
 
+def _is_state_update_variable(key, state: State):
+    if key in state.keys() and key not in TENDENCY_TO_STATE_NAME:
+        # the second check is to exclude derived variables such as dQu,v
+        return True
+    elif key == TOTAL_PRECIP_RATE:
+        # Special case where models predict precip rate which is
+        # converted to state update on accumulated precip
+        return True
+    elif key in TENDENCY_TO_STATE_NAME:
+        return False
+    else:
+        raise ValueError(
```

Comment:
The name `_is_state_update_variable` does not suggest this function raises an error, and I'm not convinced this is something that should kill the model execution. Can you delete this error or raise/make a warning in the calling routine?